### ISSUE TITLE
Duckduckgo bangs

### ISSUE THREAD

**Issue by DPAUTH** [duckduckgo/android#238]
Bangs on the duckduckgo mobile app should give the ability to search in apps. Chrome on android uses bangs to give the ability to search through apps if I want to. For example when using !maps it pops out a message that gives the ability to search it through the google maps site in chrome or in the google maps app. I would like to add that on the duckduckgo app.

**Comment by zekiel** [comment]
thanks for this @DPAUTH -- this is something I'd love as well. 

On DuckDuckHack, do you need any help with your Dropbox IA? I noticed it was closed last week or so.

**Comment by DPAUTH** [comment]
@zekiel Yeah thanks for asking! Please tell me extensively how can I take the file I need from my pull request and turn it into a new pull request? Or tell me how can I delete the file I don't need https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2701

**Comment by zekiel** [comment]
@DPAUTH  the easiest way to do it would be to ignore the closed pull request you already made. It contains 2 files and you only need to submit 1 (the JSON file). 
Since you've already done the leg work of making the JSON file, just re-do the [setup](http://docs.duckduckhack.com/welcome/setup-dev-environment.html) part, starting with forking /duckduckgo/zeroclickinfo-goodies.  
When you get to [the part where you start entering data](http://docs.duckduckhack.com/walkthroughs/programming-syntax.html#open-the-file-for-editing) into the blank json file, just copy/paste [your old json file](https://github.com/DPAUTH/zeroclickinfo-goodies/blob/131dd59955b0f99f21b21a21f6da4927eb6ebc7c/share/goodie/cheat_sheets/json/dropbox.json) and [submit a new pull request. ](http://docs.duckduckhack.com/submitting/pull-request.html). 

If you'd like, we can set up a time and walk through it next week. I'm generally available on Wed and Thursdays. It should take ~ 20 minutes or so

**Comment by DPAUTH** [comment]
@zekiel Should I also make a new IA page?

**Comment by zekiel** [comment]
@DPAUTH  yes, though, if you have any trouble please let me know. It may cause a duplication error if the ID or name is the same as a current IA page but we should be able to fix that immediately if it happens.
